Sources/Rendering/Core/Mapper/index.d.ts

```diff
@@ -291,21 +275,10 @@ export interface vtkMapper extends vtkAbstractMapper3D {
   /**
    *
    * @param scalars
+   * @param {Boolean} cellFlag
    * @param {Number} alpha
    */
-  mapScalarsToTexture(scalars: any, alpha: number): void;
-
-  /**
-   *
-   * @param {Number} scalarValue
-   * @param {Number} rangeMin
-   * @param {Number} invRangeWidth
-   */
-  scalarToTextureCoordinate(
-    scalarValue: number,
-    rangeMin: number,
-    invRangeWidth: number
-  ): IScalarToTextureCoordinate;
+  mapScalarsToTexture(scalars: any, cellFlag: boolean, alpha: number): void;
```

Comment:
looks like it is a breaking change. you might want to flag that commit as such (and update BREAKING_CHANGES.md)

Reply:
Yes, I was wondering if we really need to have this function in the publicAPI, we could simply make it private.
It would be a breaking change too though.
What do you think?

Reply:
Maybe not private but protected. I do not have a strong opinion on this. 
Please flag your commit as BREAKING_CHANGE though...